src/components/PurchaseOrder/PO.js

```diff
@@ -242,10 +241,21 @@ class PO extends Component {
     const orderNumber = get(order, 'po_number', '');
     const poLines = get(order, 'po_lines', []);
     const workflowStatus = get(order, 'workflow_status');
-    const isCloseOrderButtonVisible = orderId && workflowStatus === WORKFLOW_STATUS.open;
+    const isWorkflowStatusOpen = workflowStatus === WORKFLOW_STATUS.open;
+    const isCloseOrderButtonVisible = orderId && isWorkflowStatusOpen;
+    const isReceiveButtonVisible = orderId && isWorkflowStatusOpen && (poLines.length > 0);
```

Comment:
I think `poLines.length` is not enough. On the meeting Dennis mentioned physical quantity and electronic. So it looks like you supposed to look (maybe reduce) `poLines` to have these values more than 0

Reply:
Ok. Now physical and/or electronic quantity are required fields to fill in any case.

Reply:
The key word here `Now`. Who knows what order types could be in a future, and Dennis was clear regarding to look at quantity fields

Reply:
I got it, thank you.